Expensify/App
# Remove deleted messages from the unread message count

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/5237

### Tests | QA Steps
1. Open the App on the web with account A.
2. Send a few messages. Count the number.
3. Delete some of the recently sent messages. Count the remaining number.
4. Log out, log in with account B for the same Chat.
5. Count should be correct for unread messages.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/140608201-4fb6c060-8222-4e5a-afff-a141bd0cfc4d.mp4


#### Mobile Web


#### Desktop


#### iOS


#### Android
